interface/frontend/src/assets/css/project.css

```diff
@@ -4,4 +4,16 @@ div#version {
   top: .5rem;
   right: 1rem;
   z-index: 10000;
-}
\ No newline at end of file
+}
+
+div.container {
```

Comment:
Do we really want this global style for all div.container? the class container might be used by other developers and would then produce some unexpected behavior